src/plone/base/utils.py

```diff
@@ -36,25 +61,25 @@ def human_readable_size(size):
     return f"{float(size / float(SIZE_CONST[c])):.1f} {c}"
 
 
-def safeToInt(value, default=0):
+def safe_to_int(value, default=0):
```

Comment:
+1 
I was just to propose a name change but now it's done :D
Only, maybe we should rename it to ``safe_int`` to be closer to the other names we are using here: ``safe_text``, ``safe_callable``, etc.

Reply:
I tried to not change it as much, but want to get  rid of camel case. In fact I first had it w/o `to`. Well, I am +/- 0 for both.

Reply:
ok, changed and added deprecation, because I used the form already. Just to be sure.